packages/smart/funcs.go

```diff
@@ -744,6 +752,41 @@ func ColumnCondition(sc *SmartContract, tableName, name, coltype, permissions, i
 	return sc.AccessTable(tblName, "new_column")
 }
 
+// RowConditions checks conditions for table row by id
+func RowConditions(sc *SmartContract, tblname string, id int64, validate bool) error {
+	sql := `SELECT conditions FROM ` + converter.EscapeName(getDefTableName(sc, tblname)) + ` WHERE id = ?`
+	condition, err := model.Single(sql, id).String()
+	if err != nil {
```

Comment:
Hide sql queries in package model, please.

Reply:
Fixed in b11cae22f5cf6cf088c3834fbf84df8d91d89348